src/reverse_complement.rs

```diff
@@ -10,12 +10,11 @@ extern crate rayon;
 extern crate memchr;
 
 use std::io::{Read, Write};
-use std::{io, ptr, slice};
+use std::{cmp, io, mem};
 use std::fs::File;
 
 struct Tables {
```

Comment:
That's true: you could fill it up all the way to 256 and do `const TABLE: &[u8; 256]` it's only 4 lines 64 chars. But yeah it's a bit silly.